### PR TITLE
Update setting Info

### PR DESCRIPTION
**Pull request by zminhquanz**
Update

### CODE REVIEW

**Comment by LunaMoo** [comment]
Breaking formatting of whole file for the sake of adding 2 questionable tips? How about not doing either?:P

File: UI/GameSettingsScreen.cpp [hunk 1]
```diff
@@ -338,6 +342,10 @@ void GameSettingsScreen::CreateViews() {
 	texSecondary_->SetDisabledPtr(&g_Config.bSoftwareRendering);
 
 	CheckBox *framebufferSlowEffects = graphicsSettings->Add(new CheckBox(&g_Config.bDisableSlowFramebufEffects, gr->T("Disable slower effects (speedup)")));
+	framebufferSlowEffects->OnClick.Add([=](EventParams &e) {
+		settingInfo_->Show(gr->T("DisableSlowerEffect Tip", "Improvement performance in some scenarios on old device"));
```
Comment (unknownbrackets) [inline UI/GameSettingsScreen.cpp]: I don't think this is a good description of what it does.  Maybe, "Make special effects look bad to improve performance".  Hopefully this is already obvious by the name?

-[Unknown]

File: UI/GameSettingsScreen.cpp [hunk 2]
```diff
@@ -305,7 +305,11 @@ void GameSettingsScreen::CreateViews() {
 #endif
 
 #ifdef _WIN32
-	graphicsSettings->Add(new CheckBox(&g_Config.bVSync, gr->T("VSync")));
+	CheckBox *vsync = graphicsSettings->Add(new CheckBox(&g_Config.bVSync, gr->T("VSync")));
+	vsync->OnClick.Add([=](EventParams &e) {
+		settingInfo_->Show(gr->T("Vsync Tip", "Avoid tearing screen , always on"));
```
Comment (unknownbrackets) [inline UI/GameSettingsScreen.cpp]: Always on is probably confusing.  Perhaps, "Avoid screen tearing, depends on video driver"?  What are you trying to communicate here?

-[Unknown]

**Review by unknownbrackets — CHANGES_REQUESTED** [review]
Would like to avoid the changes to other parts of the file, but what are you trying to clarify with these settings?

I think we could potentially make Vsync longer, just "Vsync (avoid tearing)" or something, even, if needed.  Would rather avoid the tips if we don't need to use them.

-[Unknown]

**Comment by zminhquanz** [comment]
Ok , I will change this

**Comment by zminhquanz** [comment]
@unknownbrackets I was changed that as you request

**Review by zminhquanz — COMMENTED** [review]
I update info again

**Comment by unknownbrackets** [comment]
Maybe rebase it without the other formatting changes?

-[Unknown]

**Pull request closed without merging**